makefile

```diff
@@ -24,41 +25,31 @@ run-ra-dev: ## run the react-admin server.
 ########
 
 run: ## fresh run of all you need to use the app
-	make run-postgrest-docker && make create-model && make populate-db && make run-ra-dev
+	make start-supabase && make supabase-migrate-database && make supabase-populate
```

Comment:
2 things:

- `make supabase-populate` should be a manual script to run, as you don't want the script to keep adding new lines each time you start the local env
- You can go even further in the simplification of the DX, and also add `make run-ra-dev` to this target. This way you can have everything up an running with a single `make run`! (don't forget to update the Readme if you accept this change)